src/main/java/io/github/lambdatest/gradle/LambdaTestTask.java

```diff
@@ -58,20 +62,20 @@ public void runLambdaTest() {
         try {
             if (appIdFuture != null) {
                 appId = appIdFuture.join();
-                System.out.println("App uploaded successfully with ID: " + appId);
+                logger.info("App uploaded successfully with ID: {}", appId);
             }
 
             if (testSuiteIdFuture != null) {
-                testSuiteId = testSuiteIdFuture.join(); 
-                System.out.println("Test suite uploaded successfully with ID: " + testSuiteId);
+                testSuiteId = testSuiteIdFuture.join();
+                logger.info("Test suite uploaded successfully with ID: {}", testSuiteId);
             }
         } catch (CompletionException e) {
-            System.err.println("Failed to execute tasks: " + e.getMessage());
+            logger.error("Failed to execute tasks: {}", e.getMessage());
```

Comment:
Hi @subhramit,

Thank you for the PR.

You can just log full exception in here 
```suggestion
            logger.error("Failed to execute tasks: {}", e);
```